### ISSUE TITLE
Move WSL install instructions from SLiM-Extras into the manual

### ISSUE THREAD

**Issue by bhaller** [MesserLab/SLiM#456]
Hi @bryce-carson.  I've got an old note to myself to migrate the WSL install docs at https://github.com/MesserLab/SLiM-Extras/blob/master/installation/Windows10Installation.md into the SLiM manual itself.  This is still desirable, and those instructions are up to date as far as you know, yes?  If so, I'd like to resolve this for the next SLiM release.

I'm a little bit puzzled, though, because there are already WSL install instructions in the manual.  Do the instructions you wrote up supplant those earlier instructions, written by Bernard Kim?  What is the reason that you wrote new instructions in SLiM-Extras?  Presumably I knew this once, but have forgotten.

Perhaps, if you have the time, the best thing would be for you to make the manual revisions yourself?  If so, I can sent you a Pages doc for the current Chapter 2, and you can revise it with tracked changes as usual?  (BTW, I already removed all mention of AppImage installs, as I believe we decided upon a while but, but which had not yet been done.)  Or if you don't have time to do this yourself, perhaps you could just give me a little background/guidance that answers my questions above?  Thanks!

**Comment by bryce-carson** [comment]
Hi Ben,

Those instructions were written for Windows 10 before we had a
Windows-native binary, and before that other person's WSL instructions.

Regards,

Bryce.

**Comment by bhaller** [comment]
Hi @bryce-carson!  Oh, interesting.  OK, so I guess I ought to just remove them from the SLiM-Extras repo and leave it at that?

**Comment by bryce-carson** [comment]
> Hi @bryce-carson!  Oh, interesting.  OK, so I guess I ought to just remove them from the SLiM-Extras repo and leave it at that?

Yeah, as long as the instructions aren't linked to from SLiM's readme, indeed.

**Comment by bhaller** [comment]
Ah, it was referenced from the README.md file in SLiM's repo, though.  I've just updated that, removing the reference and otherwise updating the info provided there.  :->
